src/utils/retryFetch.ts

```diff
@@ -0,0 +1,35 @@
+import { LoadUmdBundleBackoff } from "./loadUmdBundle";
+
+const wait = (delay: number) =>
+  new Promise((resolve) => setTimeout(resolve, delay));
+
+export const retryFetch = async <T>({
+  loader,
+  options,
+}: {
+  loader: () => Promise<T>;
+  options: LoadUmdBundleBackoff;
```

Comment:
Super nitpick if you use this `LoadUmdBundleBackoff` here, you probably could name it `RetryOptions` or something? If feels weird to me that something generic like `retryFetch` takes in something "specific" in like `LoadUmdBundleBackoff` 👍